### ISSUE TITLE
Patch to fix OUI's in airgraph-ng

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#1123]
**Reported by swc666 on 12 May 2013 18:30 UTC**

The mac:company name dictionary is off - this fixes the OUI stuff in the generated graphs:

```
--- libOuiParse.py	2013-05-12 11:26:05.298230232 -0700
+++ libOuiParse.py.new	2013-05-12 11:25:55.320229894 -0700
@@ -143,7 +143,7 @@
             if Hex.search(line) != None:
                 lineList = Hex.search(line).group().replace("\t"," ").split("  ")
                 #return the matched text and build a list out of it
-                HexOui[= lineList[2](lineList[0].replace("-",":")])
+                HexOui[= lineList[3](lineList[1].replace("-",":")])
                 #build a dict in the format of mac:company name
         return HexOui

```

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 12 May 2013 18:30 UTC**

Ugh, can't attach tickets http://pastie.org/7899455 is the diff

**Comment by aircrack-ng** [comment]
**Attachment by misterx on 19 May 2013 23:44 UTC**

Uploaded file [fix_OUI_in_graph.diff](https://github.com/aircrack-ng/trac-attachments/blob/master/1123/20130519-234454_401-fix_OUI_in_graph.diff)

Fix OUI in graph

**Comment by aircrack-ng** [comment]
**Modified by misterx on 12 May 2013 18:30 UTC**

**Comment by aircrack-ng** [comment]
**Comment by misterx on 12 May 2013 18:30 UTC**

Fixed in #3044.
